src/lib/sql/sql_pipeline.hpp

```diff
@@ -116,13 +92,14 @@ class SQLPipeline : public Noncopyable {
   std::vector<std::shared_ptr<SQLQueryPlan>> _query_plans;
   std::vector<std::vector<std::shared_ptr<OperatorTask>>> _tasks;
   std::shared_ptr<const Table> _result_table;
+
   // Indicates whether get_result_table has been run successfully
-  bool _pipeline_was_executed = false;
+  bool _pipeline_was_executed{false};
 
   // Indicates whether translating a statement in the pipeline requires the execution of a previous statement
   // e.g. CREATE VIEW foo AS SELECT * FROM bar; SELECT * FROM foo;
   // --> requires execution of first statement before the second one can be translated
-  bool _requires_execution;
+  bool _requires_execution{false};
```

Comment:
One question: Here you use the braces but in constructors, you use parantheses. Is there a reason for this?

Reply:
because I'm bound by my own habits and I don't tend to question them ;)